cppForSwig/cryptopp/Makefile

```diff
@@ -92,9 +93,11 @@ endif
 endif
 
 ifeq ($(UNAME),Darwin)
-AR = libtool
-ARFLAGS = -static -o
-CXX = c++
+# JB: Fix this so c++ is used if compiling natively
```

Comment:
I thought c++ was something other than clang++. In any case, the logic has to be expanded upon, so that c++ is only used for native OS X builds since there is no c++ on Linux.

There were errors due to the archive stuff. We don't want to override the variables that are defined by autotools, because we want to use x86_64-apple-darwin11-* tools when cross compiling for OS X.